assets/js/main.js

```diff
@@ -390,9 +393,9 @@
 			$( '.hide-if' ).each( function( index ) {
 				var element = $( this );
 				var parent_tag = element.parent().prop( 'nodeName' ).toLowerCase()
-				
+
 				// Field.
-				if ( 'td' === parent_tag || 'label' === parent_tag ) {
+				if ( 'td' === parent_tag || 'label' === parent_tag || is_visual_field ) {
```

Comment:
Doesn't this need it's own `is_visual_field` definition? We should make this a function to avoid duplicate code.

Reply:
@jamesckemp refactored!